torii/pytests/test/health/test_health.py

```diff
@@ -0,0 +1,83 @@
+import time
+
+import requests
+import pytest
+import allure
+
+from common.settings import BASE_URL
+
+@pytest.fixture(scope="function", autouse=True)
+def setup_health_check():
+    allure.dynamic.label('endpoint', "/health")
+    allure.dynamic.label("method", "GET")
+    allure.dynamic.label("status_code", "200")
+
+@allure.id("1035")
+def test_health_status_presence(
+        GIVEN_get_request_to_health_endpoint_is_sent):
+    with allure.step(
+            "WHEN I get the response"):
+        response = GIVEN_get_request_to_health_endpoint_is_sent
+    with allure.step(
+            "THEN the response should contain health status"):
+        assert response is not None, \
+            "Response does not contain any information"
```

Comment:
Maybe that's extra too.

Reply:
I won't be removing this test for now, until the xfail issue with the normal “Healthy” check is finally resolved.